python_modules/dagster/dagster/_core/definitions/metadata/__init__.py

```diff
@@ -1253,7 +1251,7 @@ class MyMetadataEntries(NamedspacedMetadataEntries):
         return cls(**kwargs)
 
 
-class TableMetadataEntries(NamespacedMetadataEntries, frozen=True):
+class TableMetadataMixin(NamespacedMetadataMixin, frozen=True):
```

Comment:
After composing the mixin, are we sure we get another mixin? When I think of subclassing `XXXMixin`, I probably have to implement an abstractmethod to get it to work.

This is not the case for `TableMetadataMixin` -- it has no additional abstractmethods. Or are we implying that `namespace` could be overridden again?

```suggestion
class TableMetadata(NamespacedMetadataMixin, frozen=True):
```